massive-data-import/README.md

```diff
@@ -0,0 +1,152 @@
+  "trip_distance":    { "type": "double" },
+  "pickup_position":  { "type": "geo_point" },
+  "dropoff_position": { "type": "geo_point" },
+  "fare_amount":      { "type": "double" }
+}
+```
+
+## Importing data to Kuzzle
+
+The first way to import important data sets into Kuzzle is to use the [Bulk Controller](https://docs.kuzzle.io/api-documentation/controller-bulk/) [import action](https://docs.kuzzle.io/api-documentation/controller-bulk/import/).
+Its operation and syntax is similar to that of the [Elasticsearch Bulk API](https://www.elastic.co/guide/en/elasticsearch/reference/5.5/docs-bulk.html).
+
+This method is very fast but it writes almost directly in Elasticsearch. Other Kuzzle features such as [Real-Time Notifications](https://docs.kuzzle.io/guide/essentials/real-time/) will not be available.  
+
+To use it, you must go directly through a request sent to the Bulk Controller.  
+This query contains an array of objects organized by peers. The n element of the array is an object containing the name of the index and the collection and the n+1 element contains the document to insert.  
+
+```js
+const bulkQuery = {
+  body: {
+    bulkData: [
+      { index: { _index: 'nyc-open-data', _type: 'yellow-taxi' } },
+      {
+        pickup_datetime:  '03/01/2016 12:10:28 AM',
+        dropoff_datetime: '03/01/2016 12:21:56 AM',
+        passenger_count:  '1',
+        trip_distance:    '2.5',
+        pickup_position:  { lon: '-73.974746704101563', lat: '40.793346405029297' },
+        dropoff_position: { lon: '-73.979293823242187', lat: '40.761508941650391' },
+        fare_amount:      '11.8'
+      },
+
+      { index: { _index: 'nyc-open-data', _type: 'yellow-taxi' } },
+      {
+        pickup_datetime:  '03/01/2016 12:10:29 AM',
+        dropoff_datetime: '03/01/2016 12:19:05 AM',
+        passenger_count:  '5',
+        trip_distance:    '2.51',
+        pickup_position:  { lon: '-73.993759155273438', lat: '40.718101501464844' },
+        dropoff_position: { lon: '-73.951248168945312', lat: '40.711910247802734' },
+        fare_amount:      '11.3'
+      }
+    ]
+  }
+}
+```
+
+This request is then sent to the Bulk Controller with the [Kuzzle SDK](https://docs.kuzzle.io/sdk-reference/kuzzle/query/).
+
+```js
+kuzzle
+  .queryPromise({ controller: 'bulk', action: 'import' }, bulkQuery)
+  .catch(error => {
+    console.error('Error: ');
+    console.dir(error, {colors: true, depth: null})
+    process.exit(1);
+  })
+  .then(() => {
+    console.log(`${bulkQuery.body.bulkData.length / 2} lines inserted`);
+  })
+```
+
+The second method is about half as fast but it allows you to benefit from all the usual features of Kuzzle.  
+
+It uses the [Document Controller's mCreate](https://docs.kuzzle.io/api-documentation/controller-document/m-create/) action to insert multiple documents into the same query.  
+The creation of documents will send notifications to customers who have subscribed to a request corresponding to them.  
```

Comment:
When a document is created, Kuzzle will send a notification to clients that have subscribed to the document changes.